lms/templates/components/tabbed/tabbed_view.underscore

```diff
@@ -1,5 +1,5 @@
 <div class="page-content">
-    <nav class="page-content-nav" role="tablist"></nav>
-    <div class="sr-is-focusable" tabindex="-1"></div>
+    <nav class="page-content-nav"></nav>
```

Comment:
@dan-f this nav should have an aria-label too. "Teams" should be good. a screen reader will list it at Teams navigation"